argocd/argocd/Chart.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: "v2"
 name: "ArgoCD"
```

Comment:
@mcanevet Considering the chart name is used as the title for the generated docs and we don't really care about when instantiating it in the DevOps Stack, should we use human-friendly names (as done here)?

Reply:
I don't know if there is an impact or not...

Reply:
leaving it as it is for the time being